README.md

```diff
@@ -29,7 +29,7 @@ This module will try to update your dependencies based on how they are required
 - Before starting to update, all the Drupal configuration is consolidated and commited into GIT.
 - For each module / package updated the changes are commited:
   - For PHP packages, it commits the composer.lock
-  - For drupal extensions, it applies the updates, commit the configuration changed and the modified files.
+  - For Drupal extensions, it applies the updates, commits the configuration changed and the modified files. On multisites environments it will export/commit the configuration for all environments keeping them all synchronized (see paramaters).
```

Comment:
```suggestion
  - For Drupal extensions, it applies the updates, commits the configuration changed and the modified files. On multisites environments it will export/commit the configuration for all environments keeping them all synchronized (see parameters).
```